roles/openshift_node40/tasks/install.yml

```diff
@@ -1,9 +1,21 @@
 ---
+- name: Install openshift packages
+  package:
```

Comment:
Lets add
```
    async: 3600
    poll: 60
```
here so that ssh connection via bastion won't block the playbook

Reply:
@vrutkovs Thanks, I forgot I said I would do that. ptal.